docs/user/guide-intermediate.md

```diff
@@ -0,0 +1,323 @@
+
+- Syntactic scopes ([reference](README.md#syntactic-scopes)) are great for expanding the target to what you want to operate on, like the containing function.
+  - `"arg"` and `"item"`: often used to involve commas, like `"chuck item air"` or `"bring air after arg bat"` that take care of commas.
+  - `"funk"` for function.
+  - `"state"` for statement (which might span multiple lines or only part of a line).
+- Other modifiers that are like syntactic scopes in that they expand the target.
+  - [`"file"`](README.md#file) for the whole file.
+  - [`"block"`](README.md#block) for a contiguous block of non-empty lines.
+  - [`"line"`](README.md#line), often used in commands like `"chuck line"`.
+  - [`"paint"`](README.md#paint) for contiguous sequence of non-white space characters.
+  - [`"token"`](README.md#token).
+  - [`"word"`](README.md#word) allows you to refer to words within a snake_case or camelCase token.
+  - [`"char"`](README.md#char) for individual characters.
+
+And you can reuse the above scopes when using [relative/ordinal modifiers](README.md#previous--next--ordinal--number).
+Some examples:
+
+- `"[number] [scope]s"`, like `"chuck three lines row 12"` to delete rows 12-14.
+- `"[number] [scope]s backward"`, like `"chuck three lines backward row 12"` to delete rows 10-12.
+- `"[nth] [scope]"`, like `"chuck second word air"` to delete "Banana" from "appleBananaCherry".
+- `"[nth] last [scope]"`, like `"chuck last char air"` to delete "s" from "apples".
+- `"next/previous [scope]"`, like `"chuck until next token"` to delete everything from the cursor/selection to the next token.
+
+`"past"`, `"until"`, and `"and"` are good ways to build up targets.
+
+- `"chuck row 7 past row 9"` to delete rows 7 through 9.
+- `"chuck air until bat"` to delete tokens starting at the air token ending just before the bat token.
+- `"chuck row 7 and row 10"` to delete rows 7 and 10.
+
+There is also `"head"` and `"tail"` to expand a target through to the beginning or end of the line.
+`"take head air"` selects the air token through to the beginning of air's line.
+When followed by a modifier, they will expand their input to the start or end of the given modifier range.
+`"take head funk"` will select from the cursor to the start of the containing function.
+
+## Important concepts
+
+### Command structure
+
+Commands have an action and a target or two. `"chuck air"` has the action `"chuck"` and one target (the `"air"` token).
+`"bring air to bat"` has the action `"bring"` and two targets (the `"air"` and `"bat"` tokens).
+
+A target can be more than a single token or line. A target can be a [range](README.md#range-targets), like `"air past bat"` that spans from the `"air"` token to the `"bat"` token, and that is still considered _one_ target.
+A target can even be a vertical slice or list, like the list `"air and bat"`.
+Doing `"take air and bat"` will result in multiple selections and multiple cursors, but it is still one target.
```

Comment:
I do wonder how important this distinction is. We're actually not even consistent on this point internally. When the command comes from Talon to VSCode, it comes across as a single logical target, but after running our target pipeline, we call each output a target

Reply:
These statements were helpful to my understanding when I internalized them.  It also helps when reading the reference or talking to other people.